data/rooms/RoomMessages.js

```diff
@@ -50,8 +44,17 @@ const AllRoomMessages = [
     chance: 1
   },
   {
-    regex: /\bth?a?n?[xk]s?q?\b/gim,
+    //tests: https://regex101.com/r/hH5cN7/42
+    regex: /(?:^|\s)(?:(?:th(?:n[qx]|x)|t[xyq])|than[kx](?:[sxz]){0,2}(?:[uq]|y(?:ou)?)?)\b/i,
```

Comment:
This regex seems perfect. :+1: 

![Pending Test](https://img.shields.io/badge/Pending:-Test-006400.svg?style=flat-square&maxAge=2592000)